.github/actions/spell-check/line_forbidden.patterns

```diff
@@ -0,0 +1,39 @@
+# reject `m_data` as there's a certain OS which has evil defines that break things if it's used elsewhere
+# \bm_data\b
```

Comment:
This is purely a sample and can be removed.

It (and a lot of hte other content) comes from
https://github.com/check-spelling/spell-check-this/